### PR TITLE
feat: ClickHouse import automation PE-7137

### PR DESCRIPTION
**Pull request by djwhitt**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -137,6 +138,30 @@ services:
...
+    profiles:
+      - clickhouse
+    build:
+      context: .
+      dockerfile: Dockerfile.clickhouse-auto-import
+    restart: on-failure
+    volumes:
+      - ${PARQUET_DATA_PATH:-./data/parquet}:/app/data/parquet
+    environment:
+      - PARQUET_DATA_PATH:${PARQUET_DATA_PATH:-./data/parquet}
```
[the remark this answers is not in the document]
Comment (karlprieb) [reply docker-compose.yaml:155]: You're right

@djwhitt 👆

File: scripts/clickhouse-auto-import [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            elif [ "$export_status" = "errored" ]; then
+                error=$(echo "$status" | jq -r '.error')
+                echo "Export failed: $error"
+                rm -f "$parquet_dir"/*.parquet
+                exit 1
+            fi
+
+            echo "Waiting for export to complete..."
+            sleep 10
+        done
```
[the remark this answers is not in the document]
Comment (karlprieb) [reply scripts/clickhouse-auto-import:75]: Great suggestion for a follow up PR

File: docker-compose.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (djwhitt) [reply docker-compose.yaml:155]: Both work, but I'll adjust it to be consistent.

[the remark this answers is not in the document]
Comment (djwhitt) [reply docker-compose.yaml:155]: Ah, might need to be an object for the other syntax to work though.

**Pull request merged**